### PR TITLE
Add navvis files for unit tests

### PR DESCRIPTION
**Pull request by vjlux**
Add NavVis test files for testing. 

test data restored from https://github.com/microsoft/scantools/tree/main/test_data

### CODE REVIEW

File: scantools/tests/test_data/ocamlib_create_undistortion_lut_expected/lut_tiles_3x3_angles_0.npy
Comment (pablospe) [inline scantools/tests/test_data/ocamlib_create_undistortion_lut_expected/lut_tiles_3x3_angles_0.npy:1]: Are these files needed?

File: scantools/tests/test_data/navvis_files/cam/00013-cam1.jpg
Comment (pablospe) [inline scantools/tests/test_data/navvis_files/cam/00013-cam1.jpg:1]: It has 0 bytes. 
```
Unable to render rich display
Invalid image source.
```

Maybe it was intended.

**Comment by sarlinpe** [comment]
How large is this test data? I would very much avoid committing it to the repo, which would force all users to download it even though it is used only for testing. Instead, 1) add it as a git submodule or 2) zip it, add it as release artefact, and download it when running the tests.

**Comment by vjlux** [comment]
> How large is this test data? I would very much avoid committing it to the repo, which would force all users to download it even though it is used only for testing. Instead, 1) add it as a git submodule or 2) zip it, add it as release artefact, and download it when running the tests. I prefer option 2.

yes we can do option 2 and add a comment if you want test

**Comment by pablospe** [comment]
> I would very much avoid committing it to the repo, which would force all users to download it even though it is used only for testing

I agree, I was also proposing the same [here](https://github.com/microsoft/lamar-benchmark/pull/68#issuecomment-2178510460
) :)

Now my question is, will we anyways download all this data when we do `git clone` (actually twice because it was in the other PR too). Is there a way to avoid this? I think @mihaidusmanu had this problem in the past. What did you do? Other than `--depth 1` in main branch? Just curious, probably we would need to force push and rewrite the git history. Just raising the concern that not merging these commits is not enough to avoid downloading the data.

**Comment by sarlinpe** [comment]
By default `git clone` only clones the main branch, so it should not include any of these slides - especially since the branch of this PR has now been deleted.
